README.md

```diff
@@ -1,14 +1,207 @@
 install.packages("remotes")
 remotes::install_github("department-for-transport-public/tinScrape")
 ```
 
-There is only one function in the package, and you can call it without a url argument e.g.
+## What is tinScrape?
+
+The tinScrape package is made up of several functions that focuses on:
+
+* webscraping information and downloadable hyperlinks from GOV.UK
+* extracting information in Excel worksheets
+* transforming data tables into a tidy format, that is a long format
+* automating the migration of data stored in Google Cloud Storage (GCS) buckets into BigQuery (BQ)
+
+The package complements and supports the development and maintenance of the Department for Transport's currently internal-facing website - Transport in Numbers (TiN).
+
+## Functions included
+
+Currently, tinScrape is made up of the following main functions:
+
+* `extract_table_urls`
+* `gcp_to_bq`
+* `scrape_dynamic_methodology`
+* `scrape_orr_tables`
+* `utils`
+
+### extract_table_urls
+
+You can call this function using the following line:
 
 ```
 tinScrape::extract_table_urls()
 ```
+
+This function webscrapes the downloadable hyperlinks for published data tables from the Stats at DfT GOV.UK webpage. The function calls on other functions that are found in the utils.R script which are designed to extract hyperlinks that pertain to statistical outputs only (specifically the `collect_collections()`, `collect_links()` and `scrape_tables()` functions).
+
+For downloadable documents, such as Excel files, the `extract_table_urls()` function takes into account the different formats for these URLs: numbers or hexidecimal values. A combination of both are used on GOV.UK.
+
+No argument is required to use this function as the Stats at DfT webpage is hardcoded into the function in the first instance.
+
+### gcp_to_bq
+
+You can call this function using the following line:
+
+```
+tinScrape::gcp_to_bq()
+```
+
+This function takes a published data table, if it exists in GCS, extracts all the worksheets within the document, tidies the data and stores the final output in BQ. To do this, it calls on 3 other functions, created specifically for TiN:
+
+* `gcp_tables_to_list`
+* `tidied_df`
+* `create_bq_table`
+
+Further information about each of these functions are below.
+
+The argument required for the function is the name of the Excel file. If the function does not seem to work, check the GCS storage bucket to see if the Excel file's name you are trying to call matches with what is in the bucket.
+
+The `gcp_to_bq()` ensures the input - the name of the Excel file - is set to be uppercase, so the function is case insensitive.
+
+#### gcp_tables_to_list
+
+You can call this function using the following line:
+
+```
+tinScrape::gcp_tables_to_list()
+```
+
+For a given Excel document, this function searches for that document in GCS and reads it into R. Each marked up table in the document are outputted as lists in the R environment, where the name of the list is the worksheet's name.
```

Comment:
Each table is actually outputted as a data frame object in a list, and the naming is a bit more complicated